src/main/java/com/github/dockerjava/api/DockerClient.java

```diff
@@ -251,6 +256,48 @@
 
     DisconnectFromNetworkCmd disconnectFromNetworkCmd();
 
+    /**
+     * Enables swarm mode for the docker engine and creates a new swarm cluster
+     *
+     * @since 1.24
+     * @param swarmSpec the specification for the swarm
+     * @return the command
+     */
+    InitializeSwarmCmd initializeSwarmCmd(SwarmSpec swarmSpec);
```

Comment:
Sorry, missed that it can be enabled in runtime (because also thinking about standard swarm tests that people also asking), thought that it could be done only during startup.
But i fear that other standard tests that may be running in parallel may fail.

SmartMode related tests should be grouped into new group/category imho. And maybe we should create separate travis runs?